docs/how-to/programming_manual.md

```diff
@@ -146,7 +146,7 @@ For Linux developers, the link [here](https://github.com/ROCm/hip-tests/blob/dev
 
 ## HIP Graph
```

Comment:
Is it necessary to have this here? it is not much more than a reference, which shows up two or three times in the index or TOC.

Reply:
I agree, that it's not really needed here, but the programming manual is going to be overhauled and probably incorporated into the runtime api how-to soon, as it is just an amalgamation of snippets on different HIP-topics.